src/renderer/screens/accounts/AccountGridItem/index.js

```diff
@@ -51,7 +51,7 @@ class AccountCard extends PureComponent<Props> {
           style={{ display: hidden && "none" }}
           p={20}
           onClick={this.onClick}
-          data-e2e="dashboard_AccountCardWrapper"
+          data-automation-id="dashboard_AccountCardWrapper"
```

Comment:
i feel i'm repeating but imo it just should be `id` and classes (`className`), because these make sense beyond automation. the generic concept of identifying elements can be used for more than test. also for styling, and also helpful for a dev to inspect the dom. easier to write selectors for it,...

Reply:
Yes of course ! This PR is a duplicate, we had trouble with CI execution so we had to test with another PR.
All will be done in the first one 👍,  sorry for the confusion.